### PR TITLE
[Service] ml-agent interface for resource

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Implement ml-agent interface for ml-resource.

### CODE REVIEW

File: daemon/ml-agent-dbus-interface.c [hunk 1]
```diff
@@ -72,6 +74,23 @@ _get_proxy_new_for_bus_sync (ml_agent_dbus_service_type_e type, GError ** err)
...
+      for (i = 0; i < num_bus_types; ++i) {
+        g_clear_error (&_err);
+
+        mlsr =
+            machinelearning_service_resource_proxy_new_for_bus_sync (bus_types[i],
+            G_DBUS_PROXY_FLAGS_NONE, DBUS_ML_BUS_NAME, DBUS_RESOURCE_PATH, NULL,
+            &_err);
+        if (mlsr)
+          break;
+      }
```
Comment (songgot) [inline daemon/ml-agent-dbus-interface.c:90]: I wonder if there is no case where mlsr is NULL until the end of for-loop

Comment (songgot) [reply daemon/ml-agent-dbus-interface.c:90]: Thanks for the detailed explanation.

File: tests/capi/unittest_capi_service_agent_client.cc [hunk 2]
```diff
@@ -1250,6 +1250,52 @@ TEST_F (MLServiceAgentTest, model_gdbus_call_n)
...
+TEST_F (MLServiceAgentTest, resource_gdbus_call_n)
+{
+  int ret;
+  gchar *res_info = NULL;
+  GError *error = NULL;
+
+  MachinelearningServiceResource *proxy_for_resource
+      = machinelearning_service_resource_proxy_new_for_bus_sync (G_BUS_TYPE_SESSION,
+          G_DBUS_PROXY_FLAGS_NONE, "org.tizen.machinelearning.service",
+          "/Org/Tizen/MachineLearning/Service/Resource", NULL, &error);
```
Comment (songgot) [inline tests/capi/unittest_capi_service_agent_client.cc:1265]: Can this TC include "dbus-interface.h"?
If so, can you change "org.tizen.machinelearning.service" and "/Org/Tizen/MachineLearning/Service/Resource" to DBUS_ML_BUS_NAME and DBUS_RESOURCE_PATH ?

**Review by songgot — APPROVED** [review]
LGTM :+1:

File: tests/capi/unittest_capi_service_agent_client.cc (hunk 2 above)
Comment (jaeyun-jung) [reply tests/capi/unittest_capi_service_agent_client.cc:1265]: Agreed, I will add new PR for updating pre-defined macro in this testcase later.

Comment (jaeyun-jung) [reply tests/capi/unittest_capi_service_agent_client.cc:1265]: Added new PR #365 for this.

**Pull request merged**